src/actions/signupSigninActions.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Remove this line and fix the linting issues that occurs

Reply:
I have fixed this